src/Symfony/Component/Console/Output/OutputSectionReference.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Output;
+
+/**
+ * @internal
+ *
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+final class OutputSectionReference
```

Comment:
I wondering if we can remove the `final` constraint and add `@final` instead?

Reply:
The annotation would only be useful if we wanted to declare the class final after a stable release. Since it's a new class, the `final` keyword would be the way to go, imho.

Reply:
@ostrolocky This is a purely internal class, whether it's final or not should not make any difference to the user as they would never know about this class